<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_GameDetail</class>
 <widget class="QWidget" name="Form_GameDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea_gameview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_detail" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>POS</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_posx">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_posy">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
